.travis.yml

```diff
@@ -9,7 +9,7 @@ php:
 
 before_script: composer install
```

Comment:
please try to change that to

install:
  - composer self-update
  - composer install